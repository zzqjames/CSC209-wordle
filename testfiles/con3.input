g r 2
4 --g-y -gg-- aerie
c abi
c ekpy