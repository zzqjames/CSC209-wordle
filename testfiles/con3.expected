cannot_be: a b e i k p y 
must_be
[0] 
[1] 
[2] r 
[3] 
[4] 
